payment_source.go

```diff
@@ -38,6 +38,7 @@ type SourceVerifyParams struct {
 	Params
 	Customer string
 	Amounts  [2]uint8
+	Values   []string
```

Comment:
Cool. Yeah I was just thinking that if you come into this with very little information, both `Amounts` and `Values` are extremely generic names so it's really not intuitive at all that one would be meant for bank accounts and the other sources. It might be helpful to try and annotate this for users in some way.